modules/ossm-about-concepts-resources.adoc

```diff
@@ -0,0 +1,113 @@
+
+To support canary updates of the control plane, {SMProduct} includes support for multiple Istio versions. You can select a version by setting  `spec.version` to the version you would like to install, prefixed with a v. You can update to a new version just by changing this field.
+
+{SMProduct} supports two different update strategies for your control planes:
+
+`InPlace`:: {SMProduct} Operator immediately replaces your existing control plane resources with the ones for the new version.
+
+`RevisionBased`:: Uses Istio's canary update mechanism by creating a second control plane to which you can migrate your workloads to complete the update.
+
+After creating an Istio resource, {SMProduct} generates a revision name for the resource based on the `updateStrategy`, and creates a corresponding `IstioRevision`.
+
+[id="istiorevision-resource_{context}"]
+== IstioRevision resource
+
+The `IstioRevision` is a cluster-wide resource and the lowest-level API {SMProduct} provides. It is usually not created by the user, but by the operator itself. Its schema closely resembles that of the `Istio` resource - but instead of representing the state of a control plane you want to be present in your cluster, it represents a revision of that control plane.
+
+A revision of the control plane you want to be present in your cluster is an instance of Istio with a specific version and revision name, and its revision name can be used to add workloads or entire namespaces to the mesh. For example: by using the `istio.io/rev=<REVISION_NAME>` label.
+
+You can think of the relationship between the `Istio` and `IstioRevision` resources as similar to the relationship between Kubernetes' `ReplicaSet` and Pod: a `ReplicaSet` can be created by users and results in the automatic creation of Pods, which will trigger the instantiation of your containers.
+
+Similarly, users create an `Istio` resource which instructs the {SMProduct} Operator to create a matching `IstioRevision`, which then in turn triggers the creation of the Istio control plane. To do that, the {SMProduct} Operator will copy all of your relevant configuration from the `Istio` resource to the `IstioRevision` resource.
+
+[id="istiocni-resource_{context}"]
+== IstioCNI resource
+
+The lifecycle of Istio's Container Network Interface (CNI) plugin is managed separately when using {SMProduct} Operator. To install Istio's CNI plugin, you create an `IstioCNI` resource.
+
+The `IstioCNI` resource is a cluster-wide resource as it installs a `DaemonSet` that operates on all nodes of your cluster. You can select a version by setting the `spec.version` field, as you can see in the example below. To update the CNI plugin, change the version field to the version you want to install. Just like the `Istio` resource, it also has a `values` field that exposes all of the options provided in the `istio-cni` chart:
```

Comment:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b): Do not use to indicate a relative location in a document
```suggestion
The `IstioCNI` resource is a cluster-wide resource as it installs a daemon set that operates on all nodes of your cluster. You can select a version by setting the `spec.version` field, as you can see in the example that follows. To update the CNI plugin, change the version field to the version you want to install. Just like the `Istio` resource, it also has a `values` field that exposes all of the options provided in the `istio-cni` chart:
```